bears/ruby/RubySecurityBear.py

```diff
@@ -0,0 +1,20 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+
+
+@linter(executable='brakeman',
+        output_format='regex',
+        output_regex=r'L(?P<line>\d+)C(?P<column>\d+): (?P<message>.*)')
+class RubySecurityBear:
+    '""docstring for RubySecurityBear""'
+
+    LANGUAGES = {'Ruby'}
+    REQUIREMENTS = {GemRequirement('brakeman')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Security'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return '-p', filename
```

Comment:
`-p` could be included in `create_arguments` right?

Reply:
This is my 1st Bear after Tutorial Bear. So let me ask that question, can we include `-p` in `create_arguments` function? 😀

Reply:
you mean `return -p filename` right?

Reply:
yes

Reply:
https://brakemanscanner.org/docs/running/

Reply:
ohh wait let me have a look ;P

Reply:
It runs like `brakeman -p xyz` this way is what you meant right?

Reply:
Yes and xyz is our Ruby file which is to be executed.

Reply:
Same as https://github.com/coala/coala-bears/blob/master/bears/ruby/RubySyntaxBear.py

Reply:
You can wrap it in `create_argument`. ;)

Reply:
Can you give me a reference so that I can take a look at the format of `create_arguments` for my better understandings? 😄

Reply:
`
create_arguments(filename,file,cinfig_file,p='-p')
`
Like this

Reply:
`p` for parameters??

Reply:
yup ;P have a look at http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html